src/Files.App/Views/Settings/DevToolsPage.xaml

```diff
@@ -21,6 +22,14 @@
 			</ResourceDictionary.MergedDictionaries>
 
 			<converters:BoolNegationConverter x:Key="BoolNegationConverter" />
+			<wctconverters:BoolToVisibilityConverter
+				x:Key="BoolVisibilityConverter"
+				FalseValue="Collapsed"
+				TrueValue="Visible" />
+			<wctconverters:BoolToVisibilityConverter
+				x:Key="InvertedBoolVisibilityConverter"
+				FalseValue="Visible"
+				TrueValue="Collapsed" />
```

Comment:
WinUI now allows you to implicitly convert bool to visibility, so using WCT's one is unnecessary here.
As for the inverted bool converter, there's a local one.

Reply:
Actually it doesn't seem to work. When I open the page I get an `InvalidCastException` saying `Unable to cast object of type 'System.Boolean' to type 'Microsoft.UI.Xaml.Visibility'`